kimchi/src/circuits/lookup/index.rs

```diff
@@ -394,6 +394,15 @@ impl<F: PrimeField + SquareRootField> LookupConstraintSystem<F> {
                 }
 
                 //~ 6. Pad the end of the concatened table with the dummy value.
+                //     By padding with 0, we constraint the table with ID 0 to
```

Comment:
The comment/semantics makes a lot of sense to me, but I'm not 100% sure. I think an alternative (but subjectively more complicated) semantics would be to just throw an error, and let user add the fake value explicitly? 

Maybe a quick "yes" from @mrmr1993 would be really helpful.